### PR TITLE
Replace usages of testing_utilities::SolarSystem with testing_utilities::SolarSystemFactory

### PR DESCRIPTION
**Pull request by pleroy**
Not deleting testing_utilities::SolarSystem yet.

### CODE REVIEW

File: benchmarks/ephemeris.cpp [hunk 1]
```diff
@@ -217,70 +218,67 @@ void EphemerisL4ProbeBenchmark(SolarSystem::Accuracy const accuracy,
...
         {6371 * Kilo(Metre) + 100 * NauticalMile, 0 * Metre, 0 * Metre});
     Speed const earth_probe_speed =
-        Sqrt(ephemeris.bodies()[SolarSystem::kEarth]->
-                 gravitational_parameter() /
-                 earth_probe_displacement.Norm());
-    Velocity<ICRFJ2000Ecliptic> const earth_probe_velocity(
+        Sqrt(at_спутник_1_launch->massive_body(
+                 *ephemeris,
+                 SolarSystemFactory::name(SolarSystemFactory::kEarth)).
+                     gravitational_parameter() /
```
Comment (eggrobin) [inline benchmarks/ephemeris.cpp]: I hear there's a way of doing that.

File: benchmarks/чебышёв_series.cpp [hunk 2]
```diff
@@ -33,8 +33,8 @@
 
 #include "astronomy/frames.hpp"
 #include "quantities/si.hpp"
+#include "geometry/named_quantities.hpp"
```
Comment (eggrobin) [inline benchmarks/чебышёв_series.cpp]: alphabetzie

File: ksp_plugin_test/plugin_integration_test.cpp [hunk 3]
```diff
@@ -64,44 +67,46 @@ class PluginIntegrationTest : public testing::Test {
...
+          index == SolarSystemFactory::kSun
               ? nullptr
-              : std::make_unique<Index>(SolarSystem::parent(index));
+              : std::make_unique<Index>(SolarSystemFactory::parent(index));
       DegreesOfFreedom<Barycentric> const initial_state =
-          ICRFToBarycentric(solar_system_->trajectories()[index]->
-                            last().degrees_of_freedom());
+          ICRFToBarycentric(
+              solar_system_->initial_state(SolarSystemFactory::name(index)));
     // barf_cast, to be fixed when we have rvalue conversion.
```
Comment (eggrobin) [inline ksp_plugin_test/plugin_integration_test.cpp]: Not anymore. Please unbarf.

File: ksp_plugin_test/plugin_test.cpp [hunk 4]
```diff
@@ -528,43 +529,43 @@ TEST_F(PluginTest, VesselInsertionAtInitialization) {
...
+            SolarSystemFactory::name(SolarSystemFactory::kSun));
     EXPECT_THAT(
         from_parent,
         Componentwise(
             AlmostEquals(looking_glass_.Inverse()(
                 plugin_->CelestialFromParent(index).displacement()), 1, 5056),
             AlmostEquals(looking_glass_.Inverse()(
-                plugin_->CelestialFromParent(index).velocity()), 1, 1643885)));
+                plugin_->CelestialFromParent(index).velocity()),
+                1, 146492520)));
```
Comment (eggrobin) [inline ksp_plugin_test/plugin_test.cpp]: What body is doing this?

File: mathematica/integrator_plots.cpp [hunk 5]
```diff
@@ -305,32 +305,33 @@ void GenerateKeplerProblemWorkErrorGraphs() {
...
   std::vector<MassiveBody> bodies;
-  int const last_planet = SolarSystem::kMercury;
   Time const Δt_reference = 10 * Minute;
   {
-    not_null<std::unique_ptr<SolarSystem>> const solar_system =
-        SolarSystem::AtСпутник1Launch(SolarSystem::Accuracy::kMajorBodiesOnly);
-    SolarSystem::Bodies const solar_system_bodies =
-        solar_system->massive_bodies();
-    for (int i = 0; i <= last_planet; ++i) {
-      DiscreteTrajectory<ICRFJ2000Ecliptic> const& trajectory =
```
Comment (eggrobin) [inline mathematica/integrator_plots.cpp]: How did this compile?

Comment (eggrobin) [inline mathematica/integrator_plots.cpp]: Oh.

File: benchmarks/ephemeris.cpp (hunk 1 above)
Comment (pleroy) [inline benchmarks/ephemeris.cpp]: Done (and it was the only one).

File: benchmarks/чебышёв_series.cpp (hunk 2 above)
Comment (pleroy) [inline benchmarks/чебышёв_series.cpp]: Dnoe.

File: ksp_plugin_test/plugin_integration_test.cpp (hunk 3 above)
Comment (pleroy) [inline ksp_plugin_test/plugin_integration_test.cpp]: Unbarfed.

File: ksp_plugin_test/plugin_test.cpp (hunk 4 above)
Comment (pleroy) [inline ksp_plugin_test/plugin_test.cpp]: Trition, see the code change.  Its speed along z is about 60 cm/s.  Eris is the next bad one.

**Pull request merged**